src/pages/index.js

```diff
@@ -167,17 +268,36 @@ function getCardElement(data) {
     openModal(previewModal);
   });
 
+  function handleDeleteCard(cardElement, data) {
+    selectedCard = cardElement;
+    selectedCardId = data._id;
+
+    openModal(deleteModal);
+  }
+
+  function handleDeleteSubmit(evt) {
+    setButtonText(evt.submitter, true, "Delete", "Deleting...");
+    api
+      .deleteCard(selectedCardId)
+      .then(() => {
+        selectedCard.remove();
+        closeModal(deleteModal);
+      })
+      .catch(console.error);
```

Comment:
Please, return the initial button text back for the `confirmation` modal in `.finally`


```js
      .catch(console.error)
      .finally(() => {
        setButtonText(evt.submitter, false, "Delete");
      });
```